pkg/operation/shoot/shoot.go

```diff
@@ -466,6 +469,31 @@ func (s *Shoot) IsShootControlPlaneLoggingEnabled(c *config.GardenletConfigurati
 	return s.Purpose != gardencorev1beta1.ShootPurposeTesting && gardenlethelper.IsLoggingEnabled(c)
 }
 
+// IsHAControlPlaneConfigured returns true if HA configuration for the shoot control plane has been set either
+// via an alpha-annotation or ControlPlane Spec
+func (s *Shoot) IsHAControlPlaneConfigured() bool {
```

Comment:
Move this to the API helper package? Then you can re-use it in the scheduler instead of duplicating the code?

Reply:
What we use in the scheduler is `gardencorev1beta.Shoot` and what we use in `operation/shoot/shoot.go` is defined in the same package in `types.go` so cannot be re-used in the scheduler.

Reply:
You can have a helper function where you pass the `gardencorev1beta1.Shoot` and call it from here and from scheduler

Reply:
ok. Did not realize that shoot.GetInfo returns gardencorev1beta1.Shoot. Will move it

Reply:
done